authz/rbac_translator_test.go

```diff
@@ -311,3 +1004,15 @@ func TestTranslatePolicy(t *testing.T) {
 		})
 	}
 }
+
+func anyPbHelper(in map[string]interface{}, t *testing.T) *anypb.Any {
```

Comment:
When passing `testing.T` to helper functions:
- it should always be the first parameter (unless there is a context parameter as well)
- given that this is a helper function, there should also be a call to `t.Helper()`.